src/main/java/service/DaoUpdater.java

```diff
@@ -0,0 +1,20 @@
+package service;
+
+import java.util.List;
+import strategy.OperationTypes;
+
+public class DaoUpdater {
+    public static void updateData(List<String> data) {
+        boolean firstTime = true;
+        String[] splittedLine;
+        for (String line : data) {
+            splittedLine = line.split(",");
+            if (firstTime) {
+                firstTime = false;
+                continue;
+            }
```

Comment:
let's remove `firstTime` variable and find another way to skip the first line
and don't use static methods